fullscreen/api/element-ready-check-containing-iframe-manual.html

```diff
@@ -7,18 +7,18 @@
 <iframe allowfullscreen></iframe>
 <iframe allowfullscreen></iframe>
 <script>
-async_test(function(t)
-{
-    var iframes = document.getElementsByTagName("iframe");
-    trusted_request(t, iframes[0].contentDocument.body, document.body);
-    iframes[0].contentDocument.onfullscreenchange = t.step_func(function()
-    {
-        assert_equals(document.fullscreenElement, iframes[0]);
-        trusted_request(t, iframes[1].contentDocument.body, iframes[0].contentDocument.body);
-        iframes[1].contentDocument.onfullscreenchange = t.step_func_done(function() {
-            assert_equals(document.fullscreenElement, iframes[1]);
-        });
-        iframes[1].contentDocument.onfullscreenerror = t.unreached_func("fullscreenchange error");
-    });
-});
+window.onload = function() {
```

Comment:
I have no idea. Maybe worth opening an issue and looping in some other people as I'm not very familiar with the processing model.

Reply:
I filed a [Gecko bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1493878) for this. Hopefully DOM people would figure that out.

Reply:
Based on the discussion with bz, it seems the spec is clear on this case that the content document shouldn't be changed, because iframe without src shouldn't navigate after its initial handling. I've submitted #13236 for testing that.

Since this test is unrelated, I think we can have this workaround to help improve test coverage.